docs/learn-staking.md

```diff
@@ -123,6 +123,14 @@ Based on the above rewards distribution, nominators in validator pool B get more
 than those in pool A because pool A has more overall stake. Sam has staked 50 DOT in pool A, but he
 only gets 8.3 in return, whereas Kitty gets 12.5 with the same amount of stake.
 
+There is an additional factor to consider in terms of rewards. While there is no limit to the number
+of nominators a validator may have, a validator does have a limit to how many nominators to which it
+can pay rewards. In Polkadot and Kusama, this limit is currently 64. A validator with more than 64
+nominators is _oversubscribed_. When payouts occur, only the top 64 nominators as measured by amount
+of stake allocated to that validator will receive rewards. All other nominators are essentially
+"wasting" their stake - they used their nomination to elect that validator to the active stake, but
+receive no rewards in exchange for doing so.
```

Comment:
In this case, what would be the best option for nominators to do? And how what does it look like on the UI if a validator doesn't select the nominator for payout?

Reply:
It mentions what to do to avoid this in the nomination guide, this is meant as an overview.  I'm hesitant to describe _too_ specifically what it looks like in the UI here since not everyone will be using the Polkadot UI.

Reply:
Gotcha! That does make sense